packages/ui/components/editor/stylesEditor.css

```diff
@@ -42,6 +42,8 @@
   outline: 0;
   padding: 10px 10px;
   outline: none;
+  hyphens: auto !important;
+  word-wrap: break-word !important;
```

Comment:
What does these do and why is it !imporatant?

Reply:
that breaks the word as in small devices i check if the word is a bit big the whole word is shifted to another line which doesn't look good it seems there are many gaps in our paragraph(s).